### ISSUE TITLE
[circt-reduce] Does not work on Windows

### ISSUE THREAD

**Issue by youngar** [llvm/circt#4129]
It appears that all `circt-reduce` tests are failing due to being unable to execute the test script.

**Comment by seldridge** [comment]
Yes, I possibly made this worse by trying to avoid the need for a directory specification. This can likely be reverted.
